### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by utkarsh006**
- Updated README.md for better readability of users.
- Formatted lines of README.
- Shorten the README, so that users can interact with it for longer periods of time.
- Added contributors section.
- Highlighted important text.

## Screenshots

Screenshot (208)
Screenshot (211)

### CODE REVIEW

**Comment by utkarsh006** [comment]
@aadarsh-ram Kindly review.

File: README.md [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
-## Contribute
+Visit the [doc folder](./doc)
+> This documentation tries to follow as much as possible the documentation system from [Diataxis](https://diataxis.fr/).
 
-- to develop: [Setup a dev environment](./doc/introduction/setup-dev.md)
-- to translate: use [Crowdin](https://crowdin.com/project/openfoodfacts), project open food facts
+## Contributing
+
+- <b>to develop:</b> [Setup a dev environment](./doc/introduction/setup-dev.md)
+- <b>to translate:</b> use [Crowdin](https://crowdin.com/project/openfoodfacts), project open food facts
```
Comment (aadarsh-ram) [inline README.md:13]:
```suggestion
- <b>To develop:</b> [Setup a dev environment](./doc/introduction/setup-dev.md)
- <b>To translate:</b> use [Crowdin](https://crowdin.com/project/openfoodfacts), Project Open Food Facts
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,43 @@
...
 - report issues, feature requests etc.
 
 
-Wiki Page: https://wiki.openfoodfacts.org/GSOC_2022_-_Taxonomy_editor
 
-GitHub project: https://github.com/orgs/openfoodfacts/projects/28/views/1
+## Links
+
+<p>
```
Comment (aadarsh-ram) [inline README.md:20]: @utkarsh006 Please try to use the Markdown syntax everywhere and use HTML tags in places you'd need granular control.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,43 @@
...
+Taxonomies are at the 🧡 of Open Food Facts data structure which provides an editor.
 
-Docs, see [doc folder](./doc).
+![Github Repo Size](https://img.shields.io/github/repo-size/openfoodfacts/taxonomy-editor?style=for-the-badge&color=aqua)
 
-> **NOTE:**
-> This documentation tries to follow as much as possible the documentation system from [Diátaxis](https://diataxis.fr/).
 
-## Contribute
+Visit the [doc folder](./doc)
```
Comment (aadarsh-ram) [inline README.md:7]:
```suggestion
Please visit the [doc folder](./doc) for more documentation about the Taxonomy Editor.
```

File: README.md [hunk 4]
```diff
@@ -1,29 +1,43 @@
 # Taxonomy Editor
-Taxonomies are at the heart of Open Food Facts data structure - this project provides an editor
+Taxonomies are at the 🧡 of Open Food Facts data structure which provides an editor.
```
Comment (aadarsh-ram) [inline README.md:2]:
```suggestion
Taxonomies are at the 🧡 of Open Food Facts data structure. This project provides an user-friendly editor for editing taxonomies easily.
```

**Review by aadarsh-ram — CHANGES_REQUESTED** [review]
@utkarsh006 Great work! I'd like you to make a few minor changes and we'll be good to go!

**Comment by utkarsh006** [comment]
@aadarsh-ram Kindly review now. I have implemented the changes.

**Review by aadarsh-ram — APPROVED** [review]
Great work! Thanks for the contribution.

**Comment by utkarsh006** [comment]
@aadarsh-ram Welcome 🧡

**Pull request merged**